supertokens_python/recipe/multitenancy/interfaces.py

```diff
@@ -158,8 +158,8 @@ def __init__(self):
 
     @abstractmethod
```

Comment:
unrelated to this PR, but i found this type issue -> in create_or_update_tenant in multitenancy syncio and asyncio files, we should make `config` Optional as well (in node sdk, it's optional too).

Reply:
Created https://github.com/supertokens/supertokens-python/pull/377 to fix this.